js/ascendex.js

```diff
@@ -2135,6 +2137,52 @@ module.exports = class ascendex extends Exchange {
         return this.filterByArray (result, 'symbol', symbols);
     }
 
+    async modifyMarginHelper (symbol, amount, type, params = {}) {
+        await this.loadMarkets ();
+        await this.loadAccounts ();
+        const market = this.market (symbol);
+        const account = this.safeValue (this.accounts, 0, {});
+        const accountGroup = this.safeString (account, 'id');
+        amount = this.numberToString (amount);
+        const request = {
+            'account-group': accountGroup,
+            'symbol': market['id'],
+            'amount': amount,
+        };
+        const response = await this.v2PrivateAccountGroupPostFuturesIsolatedPositionMargin (this.extend (request, params));
+        //
+        // Can only change margin for perpetual futures isolated margin positions
+        //
+        //     {
+        //          "code": 0
+        //     }
+        //
+        const errorCode = this.safeString (response, 'code');
+        const status = (errorCode === '0') ? 'ok' : 'failed';
+        return {
+            'info': response,
+            'type': type,
+            'amount': amount,
+            'code': market['quote'],
+            'symbol': market['symbol'],
+            'status': status,
+        };
+    }
+
+    async reduceMargin (symbol, amount, params = {}) {
+        if (amount >= 1) {
```

Comment:
The argument to reduceMargin is actually a positive number, let's remove these checks

Reply:
With the endpoint for ascendex to reduce margin you need to enter a negative number. Should I remove these checks then convert the number to a negative in modifyMarginHelper if reduceMargin is called?

Reply:
@Dan-krm i'd say let's just accept any number (positive or negative) in both reduceMargin and addMargin and then pass that to modifyMarginHelper as is.